src/python/bot/fuzzers/libFuzzer/stats.py

```diff
@@ -261,15 +262,19 @@ def parse_performance_features(log_lines, strategies, arguments):
 
     match = LIBFUZZER_LOG_START_INITED_REGEX.match(line)
     if match:
-      stats['initial_edge_coverage'] = int(match.group(1))
-      stats['initial_feature_coverage'] = int(match.group(2))
+      stats['initial_edge_coverage'] = stats['edge_coverage'] = int(
+          match.group(1))
+      stats['initial_feature_coverage'] = stats['feature_coverage'] = int(
+          match.group(2))
+      start = True
```

Comment:
nit: `inited` or `libfuzzer_inited` seems more descriptive to me

Reply:
Done